packages/react/src/pages/planning-task-list/planning-task-list.tsx

```diff
@@ -30,15 +31,15 @@ export const PlanningTaskList = () => {
   const kanbanRef = useRef<Sortable>(null);
   const ganttRef = useRef<Gantt>(null);
 
-  const navigate = useNavigate();
-
   const [listView, kanbanView, ganttView] = listsData;
 
   const [view, setView] = useState(listView);
   const [index, setIndex] = useState(0);
   const [gridData, setGridData] = useState([]);
   const [filteredData, setFilteredData] = useState([]);
   const [loading, setLoading] = useState(true);
+  const [newTaskData, setNewTaskData] = useState(newTask);
```

Comment:
Why in the planning list page form opened with some filled values, but in the contact list page they all empty. I think we should make they all empty and remove `newTask` variable.